introduction.md

```diff
@@ -0,0 +1,2 @@
+* my name is Brian Toner, from Ireland. I am new to git, however I have used similar tools
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.